plugins/main/public/components/overview/mitre/intelligence/resources.tsx

```diff
@@ -98,13 +99,18 @@ function buildResource(label: string) {
     apiEndpoint: endpoint,
     fieldName: 'name',
     initialSortingField: 'name',
-    tableColumnsCreator: openResourceDetails => [
+    tableColumnsCreator: () => [
```

Comment:
**nitpick**: this was expecting to be called with a parameter and this was removed, but I do not see in the changes the calls to this method have removed the arguments that are unused.